### PR TITLE
Integrate incarceration map

### PR DESCRIPTION
**Pull request by nholden**
This WIP PR adds a new page to the website that incorporates the awesome map by @jonathanpascoe. I think this map is most usable when it has its own page that it can take up the full width on, and I think it's OK to hide the footer below the fold.

I tried to get this working at https://deploy-preview-155--texasjusticeinitiative.netlify.com/map/, but it didn't work because the map isn't served over HTTPS. Looks like we can [configure Cloudfront to serve the map over HTTPS](https://aws.amazon.com/premiumsupport/knowledge-center/cloudfront-https-requests-s3/).

Not sure if this page deserves its own navigation link or needs any helper copy (maybe in a dismissible modal to give the map the most possible screen real estate?), but I figured I'd get this started. Please feel free to take this PR and run with it if you have ideas!

closes https://github.com/texas-justice-initiative/website-nextjs/issues/149

### CODE REVIEW

File: pages/map.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        <MapIframe src="http://tji-map-viewer.s3-website.us-east-2.amazonaws.com/#4.77/31.67/-100.43" />
+      </React.Fragment>
+    );
+  }
+}
+
+export default Map;
+
+const MapIframe = styled.iframe`
+  height: calc(100vh - 153.375px);
```
Comment (nholden) [inline pages/map.js]: We don't explicitly set the height of the header on desktop, but Chrome calculated it at 153.375px. We should check this out in other browsers and consider defining it as a shared variable.

File: pages/map.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  width: 100vw;
+  border: 0;
+
+  @media (max-width: ${props => props.theme.medium}) {
+    height: calc(100vh - 90px);
```
Comment (nholden) [inline pages/map.js]: We define the height of the header as 90px, but maybe this should be pulled out into a variable that we can use here.

https://github.com/texas-justice-initiative/website-nextjs/blob/66bc4c897306a5fca58fce4e674d879274a93843/components/Header.js#L134-L135

Comment (haywood) [reply pages/map.js]: That would be great. Maybe just a `constants.js` file with an export like `HEADER_HEIGHT` , and we can add to it over time.

**Comment by judan** [comment]
Hey @nholden and @LayaTaal,

Dash helped me fix the CloudFront distribution and set up a subdomain for the map page. The map is now located here: https://map-viewer.texasjusticeinitiative.org/#4.65/31.67/-100.43
I've updated map.js with the correct link. You can also test it at localhost:3333/map

Looks like the tests are passing now, but still getting that ESLint error. 🤷🏽‍♀️

**Comment by laconc** [comment]
Awesome job @judan! Not sure if we want to address Nick's comments in this PR but I'd say this is ready to merge otherwise. The linting issue seems to be related to the GitHub Actions workflow itself. If this is ready to merge, we can just address that in a separate PR.

**Comment by laconc** [comment]
@jonathanpascoe, @nholden awesome job putting this together!

**Comment by nholden** [comment]
Thanks so much, @judan and @laconc! ✨

We're not linking to this page yet, but let's save that for another PR. I opened https://github.com/texas-justice-initiative/website-nextjs/issues/169 to address that.

**Pull request merged**